ntoskrnl/ke/amd64/cpu.c

```diff
@@ -357,9 +359,9 @@ KiReportCpuFeatures(IN PKPRCB Prcb)
         CpuFeatures = CpuInfo.Edx;
     }
 
-    DPRINT1("Supported CPU features: ");
+    DPRINT1("Supported CPU features:");
```

Comment:
Why this modifications with the space? (here and below)

Reply:
In the current logging the line ends with a trailing space from the last CPU-feature:
`(ntoskrnl/ke/i386/cpu.c:446) Supported CPU features: KF_V86_VIS KF_RDTSC KF_CR4 KF_CMOV KF_GLOBAL_PAGE KF_LARGE_PAGE KF_MTRR KF_CMPXCHG8B KF_MMX KF_WORKING_PTE KF_PAT KF_FXSR KF_XMMI KF_XMMI64 X86_FEATURE_PAE X86_FEATURE_APIC `

after Serges change the line should end with the C of APIC:
`(ntoskrnl/ke/i386/cpu.c:446) Supported CPU features: KF_V86_VIS KF_RDTSC KF_CR4 KF_CMOV KF_GLOBAL_PAGE KF_LARGE_PAGE KF_MTRR KF_CMPXCHG8B KF_MMX KF_WORKING_PTE KF_PAT KF_FXSR KF_XMMI KF_XMMI64 X86_FEATURE_PAE X86_FEATURE_APIC`

Sure, that is an absolute nitpick of Serge, but objectively seen Serges solution is indeed a bit better than the current state. I don't mind this proposal.